docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -0,0 +1,53 @@
+# Converting a PyTorch* Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
+
+PyTorch* framework is supported through export to ONNX\* format. A summary of the steps for optimizing and deploying a model that was trained with the PyTorch\* framework:
+
+1. [Export PyTorch model to ONNX\*](#export-to-onnx).
+2. [Configure the Model Optimizer](../Config_Model_Optimizer.md) for ONNX\*.
+3. [Convert an ONNX\* model](Convert_Model_From_ONNX.md) to produce an optimized [Intermediate Representation (IR)](../../IR_and_opsets.md) of the model based on the trained network topology, weights, and biases values.
+4. Test the model in the Intermediate Representation format using the [Inference Engine](../../../IE_DG/Deep_Learning_Inference_Engine_DevGuide.md) in the target environment via provided [sample applications](../../../IE_DG/Samples_Overview.md).
+5. [Integrate](../../../IE_DG/Samples_Overview.md) the Inference Engine in your application to deploy the model in the target environment.
+
+## Supported Topologies
+
+Here is the list of models that were tested and are guaranteed to be supported.
+It is not a full list of models that can be converted to ONNX\* and to IR.
+
+|Package Name|Supported Models|
+|:----|:----|
+| [Torchvision Models](https://pytorch.org/docs/stable/torchvision/index.html) | alexnet, densenet121, densenet161, densenet169, densenet201, resnet101, resnet152, resnet18, resnet34, resnet50, vgg11, vgg13, vgg16, vgg19 |
+| [Pretrained Models](https://github.com/Cadene/pretrained-models.pytorch) | alexnet, fbresnet152, resnet101, resnet152, resnet18, resnet34, resnet152, resnet18, resnet34, resnet50, resnext101_32x4d, resnext101_64x4d, vgg11 |
+
+**Other supported topologies**
+
+* [ESPNet Models](https://github.com/sacmehta/ESPNet/tree/master/pretrained)
+* [MobileNetV3](https://github.com/d-li14/mobilenetv3.pytorch)
+* F3Net topology can be converted using [instruction](pytorch_specific/Convert_F3Net.md)
+* QuartzNet topologies from [NeMo project](https://github.com/NVIDIA/NeMo) can be converted using [instruction](pytorch_specific/Convert_QuartzNet.md)
+* YOLACT topology can be converted using [instruction](pytorch_specific/Convert_YOLACT.md)
```

Comment:
I think that we have a rule to provide the page header in the `[]` instead of just word "instruction". For example, in the first case the line should be the following:

```
* F3Net topology can be converted using [Convert PyTorch F3Net to the Intermediate Representation](pytorch_specific/Convert_F3Net.md)
```

Reply:
It is written as "instruction" in tf docs, so I did to align with it.

Reply:
It is a bug of the TF docs :)